pkgs/applications/science/machine-learning/vowpal-wabbit/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchFromGitHub, cmake, boost169, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "vowpal-wabbit";
+  version = "8.9.2";
+
+  src =
+    fetchFromGitHub {
+      owner = "VowpalWabbit";
+      repo = "vowpal_wabbit";
+      rev = version;
+      sha256 = "0np11nvxvbxgfdlm4q07lw7zni6mm4l93bf7pj0rif7fz15k8f3l";
+      fetchSubmodules = true;
```

Comment:
It would be great if we could unvendor the dependencies.

Reply:
It is possible to unvendor this dep. I can work that into this PR.

Reply:
There are two new deps in master at the moment which may be trickier to vendor - but ill spend some time to get those sorted. Its a tradeoff between making it easier/more consistent between platforms and not being a bad citizen when packaged. You're right though, thanks for pointing it out

Reply:
I also removed `fetchSubmodules = true;` now the dep is done through `buildInputs`